include/roboptim/core/manifold-map/decorator/descriptive-wrapper.hh

```diff
@@ -100,15 +100,16 @@ namespace roboptim
   operator<<(std::ostream& o, DescriptiveWrapper<U, V>& descWrap)
   {
     o <<
-    "Displaying info about function in DescriptiveWrapper :" << "\n" <<
-    "the function is " << descWrap.fct().getName() << "\n" <<
-    "the function input size is " << descWrap.fct().inputSize() << "\n" <<
-    "the function output size is " << descWrap.fct().outputSize() << "\n" <<
-    "Displaying info about the function manifold in DescriptiveWrapper :" << "\n" <<
-    "the manifold is " << descWrap.manifold().name() << "\n" <<
-    "Is the manifold elementary ? : " <<
-    (descWrap.manifold().isElementary() ? "yes" : "no" ) << "\n" <<
-    "the manifold dimension is " << descWrap.manifold().representationDim() << "\n";
+      "Displaying info about function in DescriptiveWrapper:" << incindent <<
+      iendl << "the function is " << descWrap.fct().getName() <<
+      iendl << "the function input size is " << descWrap.fct().inputSize() <<
+      iendl << "the function output size is " << descWrap.fct().outputSize() << decindent <<
+      iendl << "Displaying info about the function manifold in DescriptiveWrapper:" << incindent <<
+      iendl << "the manifold is " << descWrap.manifold().name() <<
+      iendl << "Is the manifold elementary? " <<
+      (descWrap.manifold().isElementary() ? "yes" : "no" ) <<
+      iendl << "the manifold dimension is " << descWrap.manifold().representationDim()
+							      << decindent << iendl;
```

Comment:
I think you can remove the `iendl` here: the last CR is the responsibility of the function calling the print method (e.g. `std::cout << my_object << std::endl`). Here you would sometimes add trailing whitespaces for no reason.